Makefile

```diff
@@ -94,10 +94,16 @@ put-translations:
 linguas:
 	@${SRC_DIR}/tools/make-LINGUAS.sh 80 > ${SRC_DIR}/locale/LINGUAS
 
+lint: lint-js lint-css
+
 lint-js:
 	cd ${JS_DIR} \
 	&& npm run lint
 
+lint-css:
+	cd ${JS_DIR} \
+	&& npm run stylelint -- --config=stylelint.json "../css/**/*.css"
```

Comment:
@julen I wanted it to be more generally useful, but I'll look at what to move into scripts